service/resource/vaultrolev1/update.go

```diff
@@ -0,0 +1,34 @@
+package vaultrolev1
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange interface{}) error {
```

Comment:
Nitpick, these methods are in a different order than with delete :D

Reply:
Because it is alphabetical.

Reply:
It is not alphabetical here - https://github.com/giantswarm/cert-operator/pull/144/files#diff-625498a136974b78c4f5da9492a042f9R10

Reply:
I see. Cool. I will fix this in the next PRs.

Reply:
No stress man, I just know you like this stuff ordered <3

Reply:
I do and I will order the shit out of this man.